api/src/main/java/com/velocitypowered/api/permission/PermissionSubject.java

```diff
@@ -29,4 +32,24 @@ default boolean hasPermission(String permission) {
    * @return the value the permission is set to
    */
   Tristate getPermissionValue(String permission);
+
+  /**
+   * Gets the permission checker for the subject.
+   *
+   * @return subject's permission checker
+   */
+  default PermissionChecker getPermissionChecker() {
+    return permission -> {
+      final Tristate state = getPermissionValue(permission);
+      if (state == Tristate.TRUE) {
```

Comment:
Nit: Move this `if` block to the Velocity `Tristate` class, call it `toAdventureTriState()`.